docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -190,17 +330,248 @@ To process the received command on the server side:
+  ```
+
+**Note**
+
+This feature supports Tizen 4.0 and higher for Mobile.
+
+<a name="send_event"></a>
+## Sending and Processing A Custom Event
+
+To send a custom event to the server from the client side:
+
+1. Create the media controller server handle using the `mc_server_create()` function:
+
+   ```
+   ret = mc_server_create(&g_server_h);
+   ```
+
+2. Retrieve the client name using the `mc_server_foreach_client()` function:
+
+   ```
+   bool
+   activated_client_cb(const char *client_name, void *user_data)
+   {
+     GList **client_list = (GList **)user_data;
+
+     if (!client_list || !client_name) return FALSE;
+     *client_list = g_list_append(*client_list, g_strdup(client_name));
+
+     return TRUE;
+   }
+
+   GList *client_list = NULL;
+
+   ret = mc_server_foreach_client(g_mc_server, activated_client_cb, &client_list);
+   ```
+
+3. Send the event to the client using the corresponding `mc_server_send_custom_event()` function. Use the client name retrieved in the previous step to identify the client.
+For example, to send a your own event to the client, use the `mc_server_send_custom_event()` function with the event in the third parameter:
+
+   ```
+   char *request_id = NULL;
+
+   ret = mc_server_send_custom_event(g_server_h, client_name, "evnet1", NULL, &request_id);
+   ```
+
+4. Destroy the media controller server handle using the `mc_server_destroy()` function, when media controller server handle is no longer needed.
 
    ```
    mc_server_destroy(g_server_h);
    ```
+
+To process the received event on the client side:
+
+1. Create the media controller client handle using the `mc_client_create()` function:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+
+2. Define the callback that is invoked when the client receives the event.
+
+For example, to define a callback for a custom event:
+
+   ```
+   void
+   event_received_cb(const char* server_name, const char *request_id, const char *event, bundle *data, void *user_data)
+   {
+       dlog_print(DLOG_DEBUG, LOG_TAG, "Server Name: %s, Request id: %s, Event: %s\n", server_name, request_id, event);
+   }
+   ```
+
+3. Register the callback:
+
+   - To register a callback for a custom events, use the `mc_client_set_custom_event_received_cb()` function.
+
+For example, to register a callback for a custom event:
+
+   ```
+   ret = mc_client_set_custom_event_received_cb(g_client_h, event_received_cb, NULL);
+   ```
+
+4. Destroy the media controller client handle using the `mc_client_destroy()` function, when media controller client handle is no longer needed.
+
+   ```
+   mc_client_destroy(g_client_h);
+   ```
+
+
+<a name="send_event_reply"></a>
+To send the reply of completed custom event on the client side:
+
+1. Send the reply of completed custom event using the `mc_server_send_cmd_reply()` function with the request id of the custom event in the third parameter and the result of the custom event in fourth parameter:
```

Comment:
Would you please check `mc_server_send_cmd_reply()`?
This section is describing sending the reply on client side, but using sever side function.

Reply:
The `mc_server_send_cmd_reply` function should be replaced to `mc_client_send_event_reply`.